### PR TITLE
Revid stats

### PR DESCRIPTION
**Pull request by adamwight**

### CODE REVIEW

File: ores/scoring_systems/tests/util.py [hunk 1]
```diff
@@ -49,6 +49,7 @@ def test_scoring_system(scoring_system):
         ScoreRequest("fakewiki", [1], ["fake"],
                      injection_caches={1: {wait_time: 0.05}},
                      model_info=['version']))
+    print(response.errors)
```
Comment (Ladsgroup) [inline ores/scoring_systems/tests/util.py]: This doesn't look right, it should have some notes or something like that.

File: ores/scoring/models/rev_id_scorer.py [hunk 2]
```diff
@@ -62,6 +64,17 @@ def score(self, feature_values):
             }
         }
 
+    def quick_statistics(self):
+        '''Jam some data through to generate statistics'''
```
Comment (Ladsgroup) [inline ores/scoring/models/rev_id_scorer.py]: The docstring is incorrect and violates PEP257, it should be "Jam some data through to generate statistics."
Also name of the method is better to be a verb.

File: ores/scoring_systems/tests/util.py (hunk 1 above)
Comment (adamwight) [reply ores/scoring_systems/tests/util.py]: Thanks, I commented.

File: ores/scoring/models/rev_id_scorer.py (hunk 2 above)
Comment (adamwight) [reply ores/scoring/models/rev_id_scorer.py]: Good idea!

**Pull request merged**